manifests/dynamic-networks-controller.yaml

```diff
@@ -95,7 +95,12 @@ spec:
       serviceAccountName: dynamic-networks-controller
       containers:
         - name: dynamic-networks-controller
-          image: quay.io/maiqueb/multus-dynamic-networks-controller:latest
+          env:
+            - name: HOSTNAME
```

Comment:
Why not NODE_NAME? The purpose of this variable is to filter nodes. The fact that the name of the Node comes from the hostname is irrelevant.

Reply:
Fair enough. I'll change it.

Done.